src/backend/events.js

```diff
@@ -0,0 +1,28 @@
+import { stringify } from '../util'
+
+export function initEventsBackend (bridge, instanceMap, getInstanceName) {
```

Comment:
After taking another look, I think there is a problem with this implementation: it only patches the listeners for the instances scanned when the backend inits, so it doesn't affect instances created later (e.g. by toggling a `v-if`).

A better way could be simply patching `Vue.prototype.$emit`.